.env.sample

```diff
@@ -43,3 +43,8 @@ CSV_REPORTS_PATH = "public/report"
 APP_PORTAL_BASE_URL = "https://dev.sunbirded.org"
 
 FORM_SERVICE_URL = "http://player:3000"                                         // Base url for form search
+
+#CERTIFICATE_ISSUER_KID
+CERTIFICATE_ISSUER_KID = "1-de2ed8d1-e8d8-40a9-b1ba-7694a16a4c8d"               // This issuer Kid is used in sunbird RC end
+
+PROJECT_CERTIFICATE_ON_OFF = "ON/OFF"                                           // Project certificate enable or disable flag
```

Comment:
@VISHNUDAS-tunerlabs Why do we need this key in core service?

Reply:
@aks30 moving this key to projects service